eslint.config.mjs

```diff
@@ -0,0 +1,133 @@
+import react from 'eslint-plugin-react';
+import globals from 'globals';
+import babelParser from '@babel/eslint-parser';
+import eslint from '@eslint/js';
+
+export default [
+  {
+    ignores: [
+      '**/.vscode/',
+      '.github/',
+      'media/',
+      'static/',
+      '**/coverage/',
+      'docs/_build/',
+      '**/package-lock.json',
+      '**/specs/',
+      '**/?venv/',
```

Comment:
This does not appear to catch `.../venv/...` paths in practice.

Reply:
Is `**/*venv*/` too broad? The alternative is to list both `.venv` and `venv`, technically the former is the suggested way (using `uv`).

Reply:
Thinking back about it, `?venv` is not supposed to catch `venv`, because it needs one character 🤦🏼 

I'm guessing the options are:
- Keep as is
- List both `venv` and `.venv` (and fix all the other ignore files).

Reply:
Listing both seems like the easiest choice.